### DISCUSSION TITLE
Question about speed

### DISCUSSION THREAD

**Discussion by TomBonk** [keirf/greaseweazle#426] [General]
Hi.
I have just become an owner of version 4.1 and would like to thank the author for such a great device.

I installed it in combination with a drive from a PC, Raspberry Pi 4 and Amiberry to make it work like a real Amiga and everything works great. 
Games boot from floppy disks. I can write floppies from ADFs, but... it works 4 times slower than on a real Amiga. Also booting games from floppies is several times slower.
I also checked this on WinUAE.
Is this normal behavior ?

**Comment by keirf** [comment]
I believe the emulation has settings allowing you to choose between accuracy and speed. However you should ask the floppy bridge author Rob Smith for further details.
